proparse/src/main/java/org/prorefactor/treeparser/ContextQualifier.java

```diff
@@ -81,4 +81,14 @@ public static boolean isWrite(ContextQualifier cq) {
     }
   }
 
+  /**
+   * Is the symbol's value "referenced" in this context?
+   */
+  public static boolean isReference(ContextQualifier cq) {
+    if (cq == SYMBOL) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this if-then-else statement by a single return statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1126)